src/random-seed/random-seed.c

```diff
@@ -197,7 +197,7 @@ static int run(int argc, char *argv[]) {
                 if (r < 0)
                         log_debug_errno(r, "Failed to get machine ID, ignoring: %m");
                 else {
-                        r = loop_write(random_fd, &mid, sizeof(mid), false);
+                        r = random_write_entropy(random_fd, &mid, sizeof(mid), false);
```

Comment:
if you touch this, maybe add `/* credit= */` in friend of the 4th argument, as we do these days